### PR TITLE
[FrameworkBundle] improve errors in tests missing the BrowserKit component

### PR DESCRIPTION
**Pull request by xabbuh**
| Q             | A
| ------------- | ---
| Branch?       | 3.4
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #29696
| License       | MIT
| Doc PR        |

### CODE REVIEW

**Review by localheinz — APPROVED** [review]
👍

File: src/Symfony/Bundle/FrameworkBundle/Test/WebTestCase.php [hunk 1]
```diff
@@ -32,7 +33,12 @@ protected static function createClient(array $options = array(), array $server =
     {
         $kernel = static::bootKernel($options);
 
-        $client = $kernel->getContainer()->get('test.client');
+        try {
+            $client = $kernel->getContainer()->get('test.client');
+        } catch (ServiceNotFoundException $e) {
+            throw new \LogicException('The BrowserKit component is required to be able to create a client.', $e->getCode(), $e);
```
Comment (javiereguiluz) [inline src/Symfony/Bundle/FrameworkBundle/Test/WebTestCase.php]: Could we reword this to be similar to other error messages added recently?

```php
throw new \LogicException('You can not create the client used in functional tests if the BrowserKit component is not available. Try running "composer require symfony/browser-kit".')
```

**Review by ro0NL — APPROVED** [review]

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @xabbuh.
